docs/ref/django-admin.txt

```diff
@@ -564,6 +564,12 @@ directory. After making changes to the messages files you need to compile them
 with :djadmin:`compilemessages` for use with the builtin gettext support. See
 the :ref:`i18n documentation <how-to-create-language-files>` for details.
 
+This command doesn't mandate configured settings. However, when settings aren't
+configured, the command cannot ignore :setting:`MEDIA_ROOT`/
```

Comment:
can't ignore the :setting:`MEDIA_ROOT` and :setting:`STATIC_ROOT` directories or include :setting:`LOCALE_PATHS`.